### DISCUSSION TITLE
How to make Pango use custom ttf font?

### DISCUSSION THREAD

**Discussion by Oshel** [dlemstra/Magick.NET#1747] [Help]
Hello,

When I use `caption` keyword with a path to a font in `ReadSettings` it works like a charm:

```
using var caption = new MagickImage($"caption:{@string}", new MagickReadSettings
{
    Font = pathToFont,
});
```

But when I try to the same with `pango` it doesn't work- it chooses some default font:

```
using var caption = new MagickImage($"pango:{@string}", new MagickReadSettings
{
    Font = pathToFont,
});
```

What should I do to use 'custom' font while using pango? I need to use it because of subscript support.

Thank you in advance

**Comment by dlemstra** [comment]
Pango has it's own "engine" and only knows some of our settings. I suspect you will need to specify that in the pango `@string` in the correct markup: https://docs.gtk.org/Pango/pango_markup.html. But I don't know if they support font paths in their markup.

**Reply by Oshel** [reply]
I already tried using `"<span font='{fontPath} {fontSize}'>{text}</span>"` but without success. What does work is installing the font in the system but its crooked solution in my case.
